### ISSUE TITLE
Select component text is not accessible and not visible with hover

### ISSUE THREAD

**Issue by JoshBowdenConcepts** [prisma/lens#395]
I just happened to notice that your select component options are not accessible by color contrast and are also not visible on hover.

![image](https://user-images.githubusercontent.com/31220082/165780699-087e76ff-74fa-48d9-aa4c-6e773d6c68f4.png)

**Comment by carlagn** [comment]
InputList has a typo for dark theme.

`dar:text-gray-400` -> `dark:text-gray-400`
